geograypher/cameras/cameras.py

```diff
@@ -126,6 +132,15 @@ def get_lon_lat(self, negate_easting=True):
                 self.lon_lat = (-self.lon_lat[0], self.lon_lat[1])
 
         return self.lon_lat
+    
+    def get_camera_location(self, get_z_coordinate = False):
+        if get_z_coordinate:
+            return self.cam_to_world_transform[0:3, 3]
```

Comment:
Unless there's a specific reason, the return type should probably be consistent between the two options.